### PR TITLE
Ret 2785 - Respond to an application from Respondent

### PR DESCRIPTION
**Pull request by marcjudge**
Please remove this line and everything above and fill the following sections:

### JIRA link
https://tools.hmcts.net/jira/browse/RET-2785

### Change description
Respond to an application from Respondent

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ikirsanov — DISMISSED** [review]

File: src/main/resources/locales/cy/translation/respond-to-application.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "p1": "The respondent has applied to ",
+  "p1a": ".",
+  "p2": "If you have any objections or responses to their application you must send them to the tribunal as soon as possible and by ",
+  "p2a": " at the latest.",
+  "p3": "If you need more time to respond, you may request more time from the tribunal. If you do not respond or request more time to respond, the tribunal will consider the application without your response.",
+  "p4": "Consider in your response:",
+  "bullets": {
+    "li1": "If you agree with the application or not",
+    "li2": "If you disagree, explain why",
+    "li3": "If you want to apply for a hearing to discuss your response"
```
Comment (Itay-Kirsh) [inline src/main/resources/locales/cy/translation/respond-to-application.json:13]: Remove all English translations

File: src/main/definitions/contact-applications.ts [hunk 2]
```diff
@@ -15,5 +15,31 @@ export default [
...
+  ],
+  b: [
+    'change-details',
+    'Change personal details',
+    'reconsider-decision',
+    'Consider a decision afresh',
+    'reconsider-judgement',
+    'Reconsider judgement',
+  ],
   c: 'witness',
```
Comment (Itay-Kirsh) [inline src/main/definitions/contact-applications.ts:44]: Why are duplicating values here (like 'amend' and 'Amend response')? And why isn't every value duplicated (like 'witness' for type c)?

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

File: src/main/definitions/contact-applications.ts (hunk 2 above)
Comment (Itay-Kirsh) [reply src/main/definitions/contact-applications.ts:44]: Some of the applications are respondent applications that come from ExUI, might need some cleaning to make more sense.

File: src/main/resources/locales/cy/translation/respond-to-application.json (hunk 1 above)
Comment (marcjudge) [reply src/main/resources/locales/cy/translation/respond-to-application.json]: Fixed

File: src/main/definitions/contact-applications.ts (hunk 2 above)
Comment (marcjudge) [reply src/main/definitions/contact-applications.ts]: Split data structure in to claimant and respondent

**Review by Itay-Kirsh — APPROVED** [review]

**Review by ikirsanov — APPROVED** [review]

**Pull request merged**